README.md

```diff
@@ -1,4 +1,8 @@
-# Development Docker deployment for Personium
+# Personium Unit - Docker Container
+
+Personium [architecture](https://personium.io/docs/en/introduction/001_Personium_Architecture/) has three-layered objects. We highly recommend reading the architecture to conceptually understand various component namings and responsibilities.
+
+This repository builds a container image and runs a Personium Unit container with Docker. A Personium Unit is operated by a [Unit Manager](https://github.com/personium/app-uc-unit-manager). If you would like to develop or test a Personium Unit without running a Unit Manager, please reach out to us in [Slack community](https://personium-io.slack.com/) to populate a Unit Manager test account.
```

Comment:
```suggestion
This repository builds a container image and runs a Personium Unit container with Docker. A Personium Unit can be managed by a [Unit Manager](https://github.com/personium/app-uc-unit-manager). If you would like to develop or test a Personium Unit without building one, please reach out to us in [Slack community](https://bit.ly/Join_Personium_Slack) so that we can create demo accounts (Cells) for you.
```

Reply:
Unit Manager is just a sample GUI for managing the whole unit (for admin) or one cell (for individual), it doesn't provide the actual functions of a Personium Unit.

I also use the correct Personium Slack link for new users. The one you provide does not have info for new users to continue.